src/main.rs

```diff
@@ -287,17 +312,26 @@ fn main() {
 
 fn draw(
     state: &mut state::State,
+    animator: &mut Animator,
     background_config: &draw::BgParams,
     input_config: &draw::InputTextParams,
     list_config: &draw::ListParams,
     surface: &mut surface::Surface,
+    animations: bool,
 ) {
     use std::iter::once;
 
-    state.process_entries();
+    state.process_entries(list_config.show_default);
```

Comment:
Something wrong with encapsulation here. List config is intended to be using only inside list_view.rs